packages/spor-react/src/input/NumericStepper.tsx

```diff
@@ -107,27 +125,48 @@ export function NumericStepper({
           width={`${Math.max(value.toString().length + 1, 3)}ch`}
           visibility={!showZero && value === 0 ? "hidden" : "visible"}
           aria-live="assertive"
-          aria-label={value.toString()}
+          aria-label={
```

Comment:
Hva sjekker denne: 
 ```
ariaLabelContext.plural !== ""
              ? t(texts.currentNumberAriaLabel(ariaLabelContext.plural))
              : ""
```

Reply:
Jeg vil ikke gjøre noen endringer på eksisterende numericSteppers, kun hvis man legger til en kontekst. Den tester om man har lagt inn en kontekst og legger inn en spesifikk label på tallet som sier for eks. "chosen number of adults". Hvis ikke leser den kun opp tallet.